dowhy/gcm/config.py

```diff
@@ -15,3 +22,8 @@ def disable_progress_bars():
 def set_default_n_jobs(n_jobs: int) -> None:
     global default_n_jobs
     default_n_jobs = n_jobs
+
+
+def add_info_log_msg(msg: str, verbose: bool) -> None:
```

Comment:
This method is not doing what it says. The use of `verbose` flag here seems redundant, which you could directly control with logging level in the logging library.